src/AmpWpPlugin.php

```diff
@@ -62,6 +66,10 @@ protected function get_service_classes() {
 			'plugin_registry'                  => PluginRegistry::class,
 			'plugin_suppression'               => PluginSuppression::class,
 			'mobile_redirection'               => MobileRedirection::class,
+			'admin.google_fonts'               => GoogleFonts::class,
```

Comment:
Adding the `GoogleFonts` service here means it is always instantiated and registered, as it is not `Conditional`.

Removing it here means it is only instantiated when it is actually required as a dependency, which is what we want for this.
```suggestion
```

Reply:
Hmm, wait, I think I'm mistaken here, let me check this first.

Reply:
Yes, sorry, ignore the above, as that would mean the `register()` method won't be called when `GoogleFonts` is instantiated.

So what we need instead is to make `GoogleFonts` conditional as well with a condition of `is_admin() && ! wp_doing_ajax()` or so.

Reply:
Done in 5c5e45291.